.github/workflows/generate_sample.yml

```diff
@@ -0,0 +1,46 @@
+name: Generate the sample project
+
+on:
+  push:
+    branches:
+      - develop
+
+jobs:
+  test:
+    name: Generate the sample project
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - name: Check out
+        uses: actions/checkout@v3
+
+      - name: Set up Flutter environment
+        uses: subosito/flutter-action@v2
+        with:
+          channel: 'stable'
+          flutter-version: '3.3.10'
+
+      - name: Remove the sample project
```

Comment:
```suggestion
      - name: Remove the old sample project
```

Reply:
Fixed in https://github.com/nimblehq/flutter-templates/pull/196/commits/d4f707779738c3a4bcab3c7f0b47f609d804b0ef